CHANGELOG.md

```diff
@@ -90,8 +108,10 @@ All notable changes to the Zowe API Mediation Layer package will be documented i
 * Bugfix:  snakeyml update, scheme validation fix (#2577) ([ae48669](https://github.com/zowe/api-layer/commit/ae48669)), closes [#2577](https://github.com/zowe/api-layer/issues/2577)
 * Bugfix:  Add build info to the manifest.yaml (#2573) ([93298dd](https://github.com/zowe/api-layer/commit/93298dd)), closes [#2573](https://github.com/zowe/api-layer/issues/2573)
 * Bugfix:  Fix bug in the swagger (#2571) ([36997c6](https://github.com/zowe/api-layer/commit/36997c6)), closes [#2571](https://github.com/zowe/api-layer/issues/2571)
+* Bugfix:  Handle exceptions in extensions config reader (#2609) ([336d3b4](https://github.com/zowe/api-layer/commit/336d3b4)), closes [#2609](https://github.com/zowe/api-layer/issues/2609)
+* Bugfix:  Make the SAF IDT properties configurable in Zowe (#2610) ([b28a9dd](https://github.com/zowe/api-layer/commit/b28a9dd)), closes [#2610](https://github.com/zowe/api-layer/issues/2610)
```

Comment:
How is this a fix?

Reply:
The fix is, that the previous version does not allow configuring the APIML with `zowe.yaml` file.